.github/workflows/installer-android.yml

```diff
@@ -0,0 +1,62 @@
+name: Installers for Android
+
+on:
+  workflow_dispatch: # manually triggered
+  push:
+    tags: 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Build installers for Android
+      uses: actions/setup-node@v3
+      with:
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        node-version: 22.x
+
+    - uses: actions/setup-java@v4
+      with:
+        distribution: 'temurin' # See 'Supported distributions' for available options
+        java-version: '21'
+
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v4
+      with:
+        gradle-home-cache-includes: |
+          mobile/android/.gradle
+
+    - run: cd app/build; sh mustang-brand.sh
+    - run: cd app; yarn install
+    - run: cd lib; yarn install
+    - run: cd backend; yarn install
+    - run: cd mobile; yarn install
+    - run: cd mobile/backend; yarn install
+    - run: cd lib; yarn add bufferutil # explicitly include bufferutil otherwise it might not be included
```

Comment:
Can you remove this here and add this to the `run: cd lib; yarn install` line above, like so:
```
run: cd lib; yarn install; yarn add bufferutil # explicitly include bufferutil otherwise it might not be included
```

or (preferred, if it works):

```
run: cd lib; yarn add bufferutil; yarn install # explicitly include bufferutil otherwise it might not be included
```

If these don't work, that gives you a hint at what the problem is.

Reply:
Yes, please simply add bufferutil to package.json, assuming it doesn't break any desktop OS.